### ISSUE TITLE
download_data.py and reference issues

### ISSUE THREAD

**Issue by sarcinelliRenan** [goddoe/hide-and-seek#1]
The `download_data.py` does not seems to be downloading the data. I tried to download the dataset from the internet but it doesn't contains the `meta.pickle` file.
The paths from the scripts are also inconsistent, the download_data.py saves the files on a tmp folder, but the other scripts are referring tho the data folder.
The `load_and_test_HaS.ipynb` tries to import the `load_data_with_meta` and `load_meta` functions from `helpers` but this functions are not implemented.

**Comment by goddoe** [comment]
Hi again, @sarcinelliRenan .
I fixed the issue that you reported. However, The codes are too old for now and there are many things that needed to be refactored or update. Therefore, I'll continue update my codes. thanks!

**Comment by sarcinelliRenan** [comment]
Hello @goddoe .
I noticed that there are some "bugs", probably outdated code, and I was trying to fix it myself.
I'm tying to use the Alexnet_v2 model, but I didn't found a checkpoint file for this network, then I started training it, but the loss is stocked in 5, and the accuracy is almost the same as random. Do you have any pre-trained model to provide.
Anyway, thanks for the attention and the fixes.
